ujt/server/client.py

```diff
@@ -0,0 +1,16 @@
+import graph_structures_pb2
```

Comment:
This is just a test client used to verify server functionality/test writing client code without having to run the entire UJT.